src/shared-components/dropdown/index.tsx

```diff
@@ -30,7 +30,7 @@ type DropdownProps = {
   optionsContainerMaxHeight?: string;
   textAlign?: 'left' | 'center';
   /**
-   * The currently selected option. Can mount as `null`
+   * The currently selected option. Can mount as `undefined`
```

Comment:
there was never a default null, if value is not provided the default is undefined

Reply:
Actually if we allow passing null explicitly this comment was sort of right so I will revert this change

Reply:
Well looks like we allow null and number as well and that was not reflected in the previous version. I pushed a fix for that, still dont know who would pass null as value